content-security-policy/script-src/script-src-strict_dynamic_double_policy_honor_whitelist.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>Whitelists in a separate policy are honored with `strict-dynamic` in the script-src directive.</title>
+    <script src='/resources/testharness.js' nonce='dummy'></script>
+    <script src='/resources/testharnessreport.js' nonce='dummy'></script>
+    <script src='../support/checkSecurityPolicyViolations.sub.js' nonce='dummy'></script>
+    <!-- CSP served:
+    1) Content-Security-Policy: script-src 'strict-dynamic' 'nonce-dummy'
+    2) Content-Security-Policy: script-src 'self' 'nonce-dummy'
+    -->
+</head>
+
+<body>
+    <h1>Whitelists in a separate policy are honored with `strict-dynamic` in the script-src directive.</h1>
+    <div id='log'></div>
+
+    <script nonce='dummy'>
+        var simpleSourcedScriptRan = false;
+        async_test(function(t) {
+            var e = document.createElement('script');
+            e.src = 'simpleSourcedScript.js';
+            e.onload = t.step_func_done(function() {
+                assert_true(simpleSourcedScriptRan);
+            });
+            e.onerror = t.unreached_func('Error should not be triggered.');
+            document.body.appendChild(e);
+        }, 'Script injected via `appendChild` is allowed with `strict-dynamic` + a nonce+whitelist double policy..');
+    </script>
+
+    <script nonce='dummy'>
+        async_test(function(t) {
+            var e = document.createElement('script');
+            e.src = 'http://example.org/script.js';
```

Comment:
Rather than `example.org` (which exists, and would cause network requests if the test failed), please use something like `{{location[scheme]}}://{{domains[www2]}}:{{ports[http][0]}}` to point to one of WPT's aliases. (Note: you'll need to rename the file to `.sub.html` for that substitution to work).